Makefile

```diff
@@ -1,11 +1,11 @@
-.PHONY: check clean install source help
+.PHONY: test ci-test clean install source
 
-check:
+test:
```

Comment:
:) make users love `check`

Reply:
we agreed today that all our makefiles will have `test` target to run tests; we can easily add `check: test`, I wouldn't mind

Reply:
I don't mind either; I'm just saying that `check` is far more standard.

Reply:
thanks for heads up then! I'll open an issue for it